quickstart.md

```diff
@@ -920,6 +920,10 @@ kubectl apply -f encrypted-image-tests.yaml
 
 Testing the SEV encrypted container launch can be completed by returning to the section on how to [Verify SEV Memory Encryption](#verify-sev-memory-encryption).
 
+## Signing the encrypted/unencrypted image and enable signature verification when running the workload
+
+Please refer to <https://github.com/confidential-containers/image-rs/blob/main/docs/quickstart/image_signing.md>
+
```

Comment:
So I wonder if we need to be a bit more careful in what we are implying here with 'enable signature verification when running the workload'. IIUC, the image-rs doc explains how to sign the image and using image-rs validation that as an image-rs integration test level piece, which is good information, but in order to get signature verification support for the kubernetes workload with CoCo we'd need to do the offline_fs_kbc set-up actions (e.g. passing in the policy, and cosign pub) in the kata/CoCo guest images and we don't have any documentation of how to do that yet, so I wonder if we need to caveats on this doc until we have this?

Reply:
Yes, you're right. As you've mentioned we do not have a doc to help set up an offline-fs-kbc, maybe still leaving this PR alone until related doc is provided would be a right thing to do. How do you like?